mfa/callbacks.md

```diff
@@ -1,10 +1,10 @@
 # Two-Factor Authentication Callbacks
-For Two-Factor Authentication Callbacks, we rely on DLR's (Delivery Receipts) sent to us by Bandwidth's Messaging platform. We then take these DLR's and send them to the specified URL that was set in the `callbackUrl` field in the [initial request](./methods/code/messaging.md).
+Two-Factor Authentication Callbacks rely on DLRs (Delivery Receipts) sent to us by Bandwidth's Messaging platform. We then take these DLRs and send them to the specified URL that was set in the `callbackUrl` field in the [initial request](./methods/code/messaging.md).
```

Comment:
Looking at other parts of the doc site, it looks like we don't use 'we' and 'us' and instead say 'Bandwidth'. I'm sorry to be nit picky about this but I think it's good to be consistent.

Something like:

Two-Factor Authentication Callbacks rely on DLRs (Delivery Receipts) sent by Bandwidth's Messaging platform. The Two-Authentication platform then takes these DLRs and sends them to the specified URL...

Reply:
Yeah I was on the fence about this too. I usually think docsites try to avoid you/us language, I'll update